crates/frontend/src/pages/context_override.rs

```diff
@@ -1,77 +1,206 @@
-use crate::types::{Config, DefaultConfig, Dimension};
+use crate::providers::alert_provider::enqueue_alert;
+use crate::types::{Config, Context, DefaultConfig, Dimension};
 use crate::utils::extract_conditions;
 use futures::join;
 use leptos::*;
 use serde::{Deserialize, Serialize};
-use serde_json::{Map, Value};
+use serde_json::{json, Map, Value};
 
 #[derive(Clone, Debug, Default)]
-pub struct TableData {
+pub struct Data {
     pub context: Vec<(String, String, String)>,
-    pub overrides: Map<String, Value>,
+    pub overrides: Vec<(String, Value)>,
 }
 
-#[derive(Serialize, Deserialize, Clone, Debug)]
-struct CombinedResourceOverride {
-    config: Option<Config>,
+#[derive(Serialize, Deserialize, Clone, Debug, Default)]
+struct PageResource {
+    config: Config,
     dimensions: Vec<Dimension>,
     default_config: Vec<DefaultConfig>,
 }
 
+#[derive(Debug, Clone)]
+enum FormMode {
+    Edit,
+    Create,
+}
+
+#[component]
+fn form(
+    context: Vec<(String, String, String)>,
+    overrides: Vec<(String, Value)>,
+    dimensions: Vec<Dimension>,
+    edit: bool,
+    default_config: Vec<DefaultConfig>,
+    handle_submit: Callback<(), ()>,
+) -> impl IntoView {
+    let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
+    let (context, set_context) = create_signal(context);
+    let (overrides, set_overrides) = create_signal(overrides);
+    let dimensions = StoredValue::new(dimensions);
+
+    let on_submit = move |_| {
+        spawn_local(async move {
+            let f_context = context.get();
+            let f_overrides = overrides.get();
+            let dimensions = dimensions.get_value();
+            let result = if edit {
+                update_context(
+                    tenant_rs.get().clone(),
+                    Map::from_iter(f_overrides),
+                    f_context,
+                    dimensions,
+                )
+                .await
+            } else {
+                create_context(
+                    tenant_rs.get().clone(),
+                    Map::from_iter(f_overrides),
+                    f_context,
+                    dimensions,
+                )
+                .await
+            };
+
+            match result {
+                Ok(_) => {
+                    logging::log!("Context and overrides submitted successfully");
+                    handle_submit.call(());
+                }
+                Err(e) => {
+                    logging::log!("Error submitting context and overrides: {:?}", e);
+                }
+            }
+        });
+    };
+    view! {
+        <ContextForm
+            dimensions=dimensions.get_value()
+            context=context.get_untracked()
+            is_standalone=false
+            handle_change=move |new_context| {
+                set_context
+                    .update(|value| {
+                        *value = new_context;
+                    });
+            }
+
+            disabled=edit
+        />
+        <OverrideForm
```

Comment:
@ShubhranshuSanjeev overrides are getting created with empty override_object

Reply:
Also in this case the operator is also missing from context card
![Uploading Screenshot 2024-06-12 at 17.28.25.png…]()